lib/java/ydoc-server/src/main/java/org/enso/ydoc/server/Ydoc.java

```diff
@@ -67,6 +69,11 @@ public Builder parser(ParserPolyfill parser) {
       return this;
     }
 
+    public Builder hostAccessBuilder(HostAccess.Builder hostAccessBuilder) {
```

Comment:
- Why is this a builder? 
- Isn't just `HostAccess` instance good enough?

```suggestion
    public Builder hostAccess(HostAccess hostAccessBuilder) {
```

- without `WebEnvironment.defaultHostAccess` being public, one might need a builder
- but having both builder and `WebEnvironment.defaultHostAccess` seems unnecessary